programs/protocol/src/data/shipper.rs

```diff
@@ -1,8 +1,12 @@
 use anchor_lang::prelude::*;
 
+use crate::Name;
+
 #[account(zero_copy)]
 #[derive(Debug, Default, PartialEq, Eq)]
 pub struct Shipper {
+    pub creator: Pubkey,
```

Comment:
Who is creator?

Reply:
the creator of the shipper account, the one that is a part of seeds

Reply:
But isn't it the same as authority in all cases?

Reply:
so far yes, but doesn't have to be in the future, it allows us to add a delegate to sign transaction for another account
It's not necessary at this point in development, but decided to differentiate between signature verification and seeds